app/models/concept_exercise.rb

```diff
@@ -9,8 +9,6 @@ class ConceptExercise < Exercise
     through: :exercise_taught_concepts,
     source: :concept
 
-  delegate :instructions, to: :git
```

Comment:
We're using this for both concept and practice exercises, so I've moved it to the base Exercise class.